README.md

```diff
@@ -484,7 +484,7 @@ spec:
 Run the following command to apply Triggertemplate.
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/02_template.yaml
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/pipelines-1.15/03_triggers/03_trigger.yaml
```

Comment:
why 03_trigger.yaml ??

It should be `02_template.yaml` as you are applying triggertemplate here